data.sql

```diff
@@ -1,155 +1,312 @@
-/* Populate database with sample data. */
+INSERT INTO species (name) VALUES ('Pokemon');
+INSERT INTO species (name) VALUES ('Digimon');
 
+-- INSERT OWNERS
+INSERT INTO owners (
+  full_name,
+  age
+) VALUES (
+  'Sam Smith',
+  34
+);
+
+INSERT INTO owners (
+  full_name,
+  age
+) VALUES (
+  'Jennifer Orwell',
+  19
+);
+
+INSERT INTO owners (
+  full_name,
+  age
+) VALUES (
+  'Bob',
+  45
+);
+
+INSERT INTO owners (
+  full_name,
+  age
+) VALUES (
+  'Melody Pond',
+  77
+);
+
+INSERT INTO owners (
+  full_name,
+  age
+) VALUES (
+  'Dean Winchester',
+  38
+);
+
+INSERT INTO owners (
+  full_name,
+  age
+) VALUES (
+  'Jodie Whittaker',
+  14
+);
+
```

Comment:
The age of `Dean Winchester` is `14`, and the age of `Jodie Whittaker` is `38`. This is normal in data entry, you just swapped those 2 by mistake. No worries 😄